deployment/istio-configs/httpbin-hsm.yaml

```diff
@@ -24,7 +24,7 @@ rules:
   resources: ["certificatesigningrequests", "certificatesigningrequests/approval", "certificatesigningrequests/status"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 - apiGroups: ["certificates.k8s.io"]
-  resourceNames: [tcsissuer.tcs.intel.com/*]
+  resourceNames: ["clusterissuers.cert-manager.io/*", "tcsclusterissuer.tcs.intel.com/*", "tcsissuer.tcs.intel.com/*"]
```

Comment:
I think we should go through mTLS workload and gateway use cases on cert-manager and tcs before this configuration change.